# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/privoxy/donotusesuppliedinitscript.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Benjamin Schieder:
	we ship our own init script, thank you very much
--- privoxy-3.0.3-stable/GNUmakefile.in	2004-02-17 15:13:27.000000000 +0100
+++ privoxy-3.0.3-stable-patch/GNUmakefile.in	2005-12-14 20:02:40.000000000 +0100
@@ -1205,7 +1205,7 @@ install: CONF_DEST LOG_DEST PID_DEST che
 		$(ECHO) "** WARNING ** current install user different from configured user. Logging may fail!!" ;\
 	fi ;\
 	$(CHMOD) $(RWD_MODE) $(LOG_DEST)/logfile $(LOG_DEST)/jarfile || exit 1 ;\
-	if [ "$(prefix)" = "/usr/local" ] || [ "$(prefix)" = "/usr" ]; then \
+	if [ "$(prefix)" = "/usr/local" ] || [ "$(prefix)" = "/usr" ] && [ 1 = 0 ] ; then \
 		if [ -f /etc/slackware-version ] && [ -d /etc/rc.d/ ] && [ -w /etc/rc.d/ ] ; then \
                $(SED) 's+%PROGRAM%+$(PROGRAM)+' slackware/rc.privoxy.orig | \
                $(SED) 's+%SBIN_DEST%+$(SBIN_DEST)+' | \
@@ -1316,7 +1316,7 @@ uninstall: CONF_DEST LOG_DEST PID_DEST c
 	fi
 
 	@# init scripts and logrotate
-	@if [ "$(prefix)" = "/usr/local" ] || [ "$(prefix)" = "/usr" ]; then \
+	@if [ "$(prefix)" = "/usr/local" ] || [ "$(prefix)" = "/usr" ] && [ 1 = 0 ]; then \
 		$(ECHO) Removing $(PROGRAM) init script ;\
 		if [ -f /etc/slackware-version ] && [ -d /etc/rc.d/ ]  && [ -w /etc/rc.d/ ] ; then \
 			$(RM) /etc/rc.d/rc.privoxy ;\
